src/main/java/seedu/address/model/Model.java

```diff
@@ -30,8 +31,31 @@
      */
     Predicate<Note> PREDICATE_SHOW_NO_NOTES = unused -> false;
 
+    Predicate<Task> PREDICATE_SHOW_NO_TASKS = task -> false;
+
     Predicate<Task> PREDICATE_SHOW_ALL_TASKS = unused -> true;
 
+    Predicate<Task> PREDICATE_SHOW_DONE_TASKS = Task::getStatus;
+
+    Predicate<Task> PREDICATE_SHOW_NOT_DONE_TASKS = task -> !task.getStatus();
+
+    Predicate<Task> PREDICATE_SHOW_OVERDUE_TASKS = new Predicate<Task>() {
+        @Override
+        public boolean test(Task task) {
+            LocalDateTime now = LocalDateTime.now();
+            System.out.println(now.toString());
+            if (task.getStatus()) {
+                return false;
+            }
+            LocalDateTime taskDateTime = LocalDateTime.of(task.getDate(), task.getTime());
+            if (taskDateTime.compareTo(now) < 0) {
+                return true;
+            } else {
+                return false;
+            }
```

Comment:
These five lines can be replaced with `return taskDateTime.compareTo(now) < 0;`.